rules-threat-hunting/windows/process_creation/proc_creation_win_regsvr32_execution.yml

```diff
@@ -0,0 +1,34 @@
+title: Export Function 'DllRegisterServer' Called - by Regsvr32
+id: ce2c44b5-a6ac-412a-afba-9e89326fa972
+related:
+    - id: 0ba1da6d-b6ce-4366-828c-18826c9de23e
+      type: similar
+status: experimental
+description: Detects when a specific sequency of arguments is called in the commandline by Regsvr32, which implicitly calls the DLL export function 'DllRegisterServer'. The DLL is located in a non-standard path. Executables are natively found in C:\Windows\system32 and C:\Windows\syswow64.
+references:
+    - https://thedfirreport.com/2023/08/28/html-smuggling-leads-to-domain-wide-ransomware/
+    - https://www.virustotal.com/gui/file/288fc4f954f98d724e6fab32a89477943df5c0e9662cb199a19b90ae0c63aebe/detection
+    - https://learn.microsoft.com/en-us/windows/win32/api/olectl/nf-olectl-dllregisterserver
+author: Andreas Braathen (mnemonic.io)
+date: 2023/09/01
+tags:
+    - attack.execution
+    - attack.t1218
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_image:
+        - Image|endswith: '\regsvr32.exe'
+        - OriginalFileName: 'REGSVR32.EXE'
+    selection_cmdline:
+        CommandLine|contains: ' /s ""'
```

Comment:
I still don't understand the logic behind this empty strings. I spend some time looking for an example but couldn't find one leveraging an empty string. If the empty string is there how is the DLL provided? 

Can you provide an example CLI of what you found in the wild?

The VT link you provided only links to a DLL with the `DllRegisterServer` export.

Reply:
cc @netgrain any info you can provide?

Reply:
Your right @nasbench. 

Minor double quotes typo, which now is revised per commit to capture all subsequent pathing.